tests/integration/security/testdata/rbac/v1beta1-jwt.yaml.tmpl

```diff
@@ -49,4 +50,11 @@ spec:
     when:
     - key: request.auth.claims[groups]
       values: ["group-2"]
+  - to:
+    - operation:
+        paths: ["/tokenAny"]
+        methods: ["GET"]
+    from:
+    - source:
+        requestPrincipals: ["*"]
```

Comment:
Add another requestPrincipals with empty string and check if requests are getting denied?

Reply:
This should be covered by the test case that doesn't use JWT token: https://github.com/istio/istio/pull/18316/files#diff-9ce4bdaa765b4a2bbcc13b50f9a6635fR139 or are you referring to something else here?

Reply:
I'm still confused why do we want to test `""`, is that related to this PR?

Reply:
Oops, I got mixed up between * and "".  `requestPrincipals: ["*"]` looks good. Resolved.